Formula/p/postgis.rb

```diff
@@ -49,10 +49,6 @@ class Postgis < Formula
     depends_on "gettext"
   end
 
-  on_linux do
-    depends_on "llvm@18" # align with `apache-arrow`
-  end
```

Comment:
Need to re-add as it does link. I thought there wasn't any in current bottle but may have missed it. Will check once CI finishes.

Reply:
I think this is due to `llvm` being linked while `llvm@18` is keg-only.

Previous bottle uses GCC libs:
```
  NEEDED       libstdc++.so.6
```

Now using LLVM libs:
```
  NEEDED       libc++.so.1
  NEEDED       libc++abi.so.1
  NEEDED       libunwind.so.1
```